libs/ballot-interpreter/src/hmpb-rust/timing_marks/corners/border_finding.rs

```diff
@@ -37,15 +37,21 @@ impl BallotGridBorders {
     ) -> Result<Self, Error> {
         let (top_left, top_right, bottom_left, bottom_right) = corners.corner_marks();
 
+        let horizontal_timing_mark_center_to_center_distance =
+            geometry.horizontal_timing_mark_center_to_center_pixel_distance();
+        let maximum_timing_mark_center_distance_error = geometry.timing_mark_width_pixels() / 2.0;
```

Comment:
Should the error bound for timing marks be relative to the expected distance between marks, rather than width of a mark?

Reply:
Yeah, that makes sense. I'll fix.